Utilities/bootstrap

```diff
@@ -198,6 +202,8 @@ def parse_build_args(args):
     args.bootstrap_dir = os.path.join(args.target_dir, "bootstrap")
     args.conf = 'release' if args.release else 'debug'
     args.bin_dir = os.path.join(args.target_dir, args.conf)
+    args.bootstrap = not args.skip_cmake_bootstrap or \
+                     not os.path.exists(os.path.join(os.path.split(args.swiftc_path)[0], "swift-build"))
```

Comment:
As a refinement, would it make sense to emit an error if `--skip-cmake-bootstrap` is true but there's no `swift-build`, rather than going through with the bootstrap?

Reply:
I figured it's better to then try the bootstrap build as a backup, but could go either way. If you prefer, I can add a warning or error out in my next pull to cross-compile SPM for Android.